### DISCUSSION TITLE
Why is 1.10.12 much slower than 1.10.10?

### DISCUSSION THREAD

**Discussion by doowhtron** [apache/airflow#12580] [Q&A] [answered]
Recently I have my airflow upgraded from 1.10.10 to 1.10.12 and find my dags are scheduled much slower than before.  One of my dag is as follows（attachment） and it has 400 Tasks of DummyOperator. It took only 4+mins with 1.10.10 while it took more than 16mins with 1.10.12.

Can someone tell me what the problem may be？I tried to compare  the code diff between these two versions with [https://github.com/apache/airflow/compare/1.10.10...1.10.12](url) and failed to draw a conclusion.

![企业微信截图_16061842899315](https://user-images.githubusercontent.com/4937178/100040745-d4552080-2e42-11eb-802e-2e23f412bafa.png)
![企业微信截图_160618431847](https://user-images.githubusercontent.com/4937178/100040754-d8813e00-2e42-11eb-9e10-1d9555fb7243.png)

[huge_dag_demo.py.py.zip](https://github.com/apache/airflow/files/5587084/huge_dag_demo.py.py.zip)

**Comment by turbaszek** [comment]
@doowhtron can you tell us more about your deployment?

**Reply by doowhtron** [reply]
Thanks for your attention. My deployment is as follows:

Host:  **2 TencentCloud VM, 8 cores, 16GB**
OS: **centos7**
kernel: **3.10.0**
Mysql: **5.7.17**

Host1 runs: **webserver, scheduler, mysqld, worker**
Host2 runs: **webserver, worker, RabbitMQ**

I found two suspicious phenomena:
1. In logs/scheduler/latest/huge_demo3.py.log, Process file took more seconds in 1.10.12 than in 1.10.10.
2. Mysqld's CPU is higher in 1.10.12 than in 1.10.10.

If you need more infomation, please let me know.

**Comment by doowhtron** [comment, marked as answer]
Problem solved by upgrading to 1.10.13

https://github.com/apache/airflow/pull/11010
